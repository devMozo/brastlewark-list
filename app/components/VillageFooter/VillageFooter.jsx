import React from 'react';

export const VillageFooter = () => {
  return (
    <footer className="VillageFooter padding-2-y margin-2-top">
      <h5> by Nicolas Mozo </h5>{' '}
    </footer>
  );
};
