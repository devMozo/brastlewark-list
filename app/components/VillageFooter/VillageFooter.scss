.VillageFooter {
  border-top: 1px solid $orange;
  text-align: center;
  color: #aaa;
}
