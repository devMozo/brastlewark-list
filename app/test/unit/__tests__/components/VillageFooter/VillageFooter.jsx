import React from 'react';
import { VillageFooter } from 'components/VillageFooter/VillageFooter';
import { render } from 'enzyme';

describe('<VillageFooter />', () => {
  let villageFooter;

  beforeEach(() => {
    villageFooter = () => {
      return render(<VillageFooter />);
    };
  });
  it('Snapshotesting <VillageFooter />', () => {
    expect(villageFooter()).toMatchSnapshot();
  });
});
