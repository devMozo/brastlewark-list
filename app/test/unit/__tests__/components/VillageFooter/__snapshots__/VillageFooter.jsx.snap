// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VillageFooter /> Snapshotesting <VillageFooter /> 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "VillageFooter padding-2-y margin-2-top",
    },
    "children": Array [
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "data": " by Nicolas Mozo ",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "h5",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "data": " ",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "text",
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
      Object {
        "data": " ",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": " by Nicolas Mozo ",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "h5",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "type": "text",
      },
    ],
    "name": "footer",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
